test/acceptance/framework/k8s/deploy.go

```diff
@@ -132,26 +120,21 @@ func CheckStaticServerConnectionMultipleFailureMessages(
 
 // CheckStaticServerConnectionSuccessful is just like CheckStaticServerConnection
 // but it always expects a successful connection.
-func CheckStaticServerConnectionSuccessful(t *testing.T, options *k8s.KubectlOptions, deploymentName string, curlArgs ...string) {
+func CheckStaticServerConnectionSuccessful(t *testing.T, options *k8s.KubectlOptions, curlArgs ...string) {
```

Comment:
Removing deploymentName arg here because it's always the same. Since the function is specific to static-server, I think it's safe to assume deployment name is static-client.